css/general.css

```diff
@@ -0,0 +1,23 @@
+/* You can introduce different colors if needed */
+:root {
+    --background-main-color: rgb(255, 255, 242);
+    --background-secondary-color: rgb(0, 37, 38);
+    --button-color: rgb(0, 135, 103);
+    --button-optional-color: rgb(251, 255, 54);
+    --button-secondary-color: rgb(216, 216, 216);
+    --text-main-color: black; 
+    --text-secondary-color: white;   
+}
```

Comment:
Might as well use the same format for black and white instead of just using their names. So
black > rgb(0, 0, 0)
white > rgb(255, 255, 255)

Reply:
Also, I think you can move the color variables to their own .css file.

Reply:
> Might as well use the same format for black and white instead of just using their names. So black > rgb(0, 0, 0) white > rgb(255, 255, 255)

Sure I can do that. 

> Also, I think you can move the color variables to their own .css file.

No need. We're gonna use them throughout all of the pages. If there're gonna be exceptions we can just make those variables on the page's css files.